web_ui/frontend/components/DataExportTable.tsx

```diff
@@ -413,7 +413,7 @@ export const DataExportTable = ({ boxProps }: { boxProps?: BoxProps }) => {
       {from_registry && pending &&
         <Box display={"flex"} flexDirection={"column"}>
           <LinearProgress sx={{mb:1, w: "100%"}} />
-          <Typography variant={'subtitle2'} color={grey[400]} mx={"auto"}>Checking Registry for Updates</Typography>
+          <Typography variant={'subtitle2'} color={grey[400]} mx={"auto"}>Starting Update Watcher; Will Poll Every 10 Seconds;</Typography>
```

Comment:
Maybe mentioning Registry here would be more straightforward?
like `Polling from Registry for Updates Every 10 Seconds`